source/dashboards/header-footer.markdown

```diff
@@ -40,6 +40,10 @@ double_tap_action:
   required: false
   description: Action taken on double tap. See [action documentation](/dashboards/actions/#double-tap-action).
   type: map
+alt_text:
```

Comment:
Can we move this above the actions?  Makes more sense right after the image source.